performance-tests/pom.xml

```diff
@@ -104,6 +104,12 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
```

Comment:
You definitely don't want to turn code coverage on for performance tests. You should consider skipping acceptance tests, really anything other than unit-tests.